lighthouse-core/config/perf.json

```diff
@@ -28,6 +28,26 @@
     "dobetterweb/script-blocking-first-paint"
   ],
 
+  "categories": {
+    "performance": {
+      "name": "Performance",
+      "description": "These encapsulate your app's performance.",
+      "audits": [
+        {"id": "first-meaningful-paint", "weight": 5},
```

Comment:
is it expected these weights don't match the weights in the existing aggregations?